include/measurement_kit/common/error.hpp

```diff
@@ -98,7 +98,14 @@ MK_DECLARE_ERROR(25, EvbufferDrain, "");
 MK_DECLARE_ERROR(26, EvbufferRemoveBuffer, "");
 MK_DECLARE_ERROR(27, EvbufferPullup, "");
 
-MK_DECLARE_ERROR(28, Type, "");
+MK_DECLARE_ERROR(28, EvdnsBaseNewException, "");
+MK_DECLARE_ERROR(29, EvdnsBaseResolveIpv4Exception, "");
+MK_DECLARE_ERROR(30, EvdnsBaseResolveIpv6Exception, "");
+MK_DECLARE_ERROR(31, EvdnsBaseResolveReverseIpv4Exception, "");
```

Comment:
Naming convention comment: I'd not append `Exception` to the error name. Instead, I am starting to think that `MK_DECLARE_ERROR` should do less magic with the error name.